src/components/util/Bubble.tsx

```diff
@@ -22,21 +22,29 @@ import KeyCodes from '../../util/KeyCodes';
 import '../../styles/bubble.scss';
 
 interface Props {
+	id?: number;
 	className?: string;
 	size?: 'small' | 'medium' | 'large';
 	style?: React.CSSProperties;
 	removeIconType?: 'default' | 'white';
 	value: string;
+	selectNext?: () => void | null;
+	selectPrev?: () => void | null;
 	isValid?: boolean;
 	autocompleteVariants?: string[] | null;
 	submitKeyCodes?: number[];
 	onSubmit?: (nextValue: string) => void;
 	onRemove: () => void;
 }
 
-export default function Bubble(props: Props) {
+export type BubbleRef = { focus: () => void };
+
+const Bubble = React.forwardRef<BubbleRef, Props>((props, ref) => {
 	const {
+		id,
```

Comment:
we dont need id, only one bubble can be focused to we can just call selectNext, selectPrev